kolibri/core/assets/src/views/app-bar.vue

```diff
@@ -209,14 +209,14 @@
   }
 
   // Will display icon in app bar if variables are defined
-  .app-bar-title-icon {
-    background: $app-bar-title-icon;
-    height: $app-bar-title-icon-height;
-    width: $app-bar-title-icon-height;
-    display: inline-block;
-    vertical-align: middle;
-    background-size: cover;
-  }
+  // .app-bar-title-icon {
+  //   background: $app-bar-title-icon;
+  //   height: $app-bar-title-icon-height;
+  //   width: $app-bar-title-icon-height;
+  //   display: inline-block;
+  //   vertical-align: middle;
+  //   background-size: cover;
+  // }
```

Comment:
was this just used in VF 0.6?

we can probably delete it, assuming we'll need to re-implement in the future using another mechanism

Reply:
Yes it was a hack to get the logo to appear in the app-bar. But in SCSS, undefined variables are not allowed.